releng/org.eclipse.xtext.redist.feature/feature.xml

```diff
@@ -31,6 +31,7 @@ http://www.eclipse.org/legal/epl-v10.html Description here.
       <import plugin="com.google.inject" version="3.0.0" match="greaterOrEqual"/>
       <import plugin="org.objectweb.asm" version="7.1.0" match="compatible"/>
       <import plugin="javax.inject" version="1.0.0" match="greaterOrEqual"/>
+      <import plugin="io.github.classgraph" version="4.8.35" match="compatible"/>
```

Comment:
aint that needed for all features?

Reply:
I thought redist was included in the others. Let me check.

Reply:
Updated. Does that make more sense?